docs/secure-connections/sauce-connect/system-requirements.md

```diff
@@ -15,12 +15,284 @@ Keep in mind that these are guidelines. Most environments have their own workloa
+
+<p><small><span className="sauceGold">Coming Soon</span></small></p>
+
+<table>
+  <tr>
+   <td><strong>Name</strong>
+   </td>
+   <td><strong>Value</strong>
+   </td>
+   <td><strong>CPU Arch</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Linux
+   </td>
+   <td>Ubuntu 18.04 or newer
+        <p>Debian 9 or newer</p>
+        <p>CentOS/RHEL-7 or newer</p>
+        <p>SUSE Linux Enterprise 15.0 and newer</p>
+   </td>
+   <td>x86_6
+       <p>arm64</p>
+   </td>
+   <td>Unsupported OSes may still be able to run Sauce Connect, but they are not tested for versions v4.8.x and higher.
+   </td>
+  </tr>
+  <tr>
+   <td>macOS
+   </td>
+   <td>OS X 10.15 (Catalina) and newer
+   </td>
+   <td>x86_64
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Windows
+   </td>
+   <td>Windows 10 and newer
+   </td>
+   <td>x86
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+:::note macOS Catalina 10.15+ compatibility
+Enable your Mac to allow apps from the App Store and identified developers.  
+  1. Go to **System Preferences** > **Security & Privacy** > **General**.
+  2. Under **Allow apps downloaded from**, select the option **App Store and identified developers**.
+:::
+
+### Sauce Connect v4.7.x
+
+<table>
+  <tr>
+   <td><strong>Name</strong>
+   </td>
+   <td><strong>Value</strong>
+   </td>
+   <td><strong>CPU Arch</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Linux
+   </td>
+   <td>Ubuntu 14.04 or newer
+        <p>Debian 8 or newer</p>
+        <p>CentOS/RHEL-6 or newer</p>
+        <p>SUSE Linux Enterprise 12.0 and newer</p>
+   </td>
+   <td>x86_64
+   </td>
+   <td>Unsupported OSes may still be able to run SauceConnect but they are not tested for Sauce Connect Proxy v4.7.x and newer. Ended support for 32b.
+   </td>
+  </tr>
+  <tr>
+   <td>macOS
+   </td>
+   <td>OS X 10.11 (El Capitan) and newer
+   </td>
+   <td>x86_64
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Windows
+   </td>
+   <td>Windows 7 and newer
+   </td>
+   <td>x86
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+
+### Sauce Connect v4.6.x
+
+<table>
+  <tr>
+   <td><strong>Name</strong>
+   </td>
+   <td><strong>Value</strong>
+   </td>
+   <td><strong>CPU Arch</strong>
+   </td>
+   <td><strong>Notes</strong>
+   </td>
+  </tr>
+  <tr>
+   <td>Linux
+   </td>
+   <td>Ubuntu 14.04 or newer
+        <p>Debian 7 or newer</p>
+        <p>RedHat EL-6 and newer</p>
+   </td>
+   <td>x86_64
+       <p>x86</p>
+   </td>
+   <td>Unsupported OSes may still be able to run SauceConnect but they are not tested for Sauce Connect Proxy v4.6.x and newer.
+   </td>
+  </tr>
+  <tr>
+   <td>macOS
+   </td>
+   <td>OS X 10.6 (Snow Leopard) or newer
+   </td>
+   <td>x86_64
+   </td>
+   <td>
+   </td>
+  </tr>
+  <tr>
+   <td>Windows
+   </td>
+   <td>Windows 7 and newer
+   </td>
+   <td>x86
+   </td>
+   <td>
+   </td>
+  </tr>
+</table>
+
+
+### Tunnel Connection TLS Specs
```

Comment:
This part and below should be in another doc. Sorry for the confusion, I meant to separate them and forgot. This would be a part of "SC specification", maybe in "Advanced" section

Reply:
fixed!